queries.sql

```diff
@@ -8,4 +8,11 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attemps FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name = 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT COUNT(name) FROM animals;
+SELECT COUNT(escape_attempts) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT name FROM animals WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);
```

Comment:
- [ ] Kindly add a query for animals who escapes the most, neutered or not neutered animals?

Reply:
Hi, the query is on line 15, it returns the name of the animal that escapes the most as seen on the fifth screenshot attached to this pull request.